package.json

```diff
@@ -26,7 +26,10 @@
     "test-grep": "mocha --bail --grep"
   },
   "dependencies": {
-    "discordie": "0.6.5"
+    "discordie": "0.6.5",
+    "express"    : "latest",
+    "request"    : "latest",
+    "cheerio"    : "latest"
```

Comment:
Using the command `npm install --save cheerio`, npm will explicitly put the version number here. See how `discordie` is set up above.

Shouldn't use `latest` here because a new version of a package might break something we're doing.